drivers/adc-dac/ad74416h/ad74416h.c

```diff
@@ -914,18 +926,25 @@ int ad74416h_init(struct ad74416h_desc **desc,
 
 	no_os_crc8_populate_msb(_crc_table, AD74416H_CRC_POLYNOMIAL);
 
-	ret = ad74416h_reset(descriptor);
+	ret = no_os_gpio_get_optional(&descriptor->reset_gpio,
+				      init_param->reset_gpio_param);
 	if (ret)
 		goto comm_err;
 
+	ret = ad74416h_reset(descriptor);
```

Comment:
Might be a good idea to also reset the ad74416h when calling the remove() function. This would leave the device in a known state.